yb-voyager/src/tgtdb/target_db_interface.go

```diff
@@ -48,7 +48,7 @@ type TargetDB interface {
 	EnableGeneratedByDefaultAsIdentityColumns(tableColumnsMap map[string][]string) error
 	GetTableToUniqueKeyColumnsMap(tableList []string) (map[string][]string, error)
 	ClearMigrationState(migrationUUID uuid.UUID, exportDir string) error
-	InvalidIndexExists(indexName string) bool
+	InvalidIndexes() (map[string]bool, error)
```

Comment:
It would be better to return a slice of index names (`[]string`) instead of a map. You can always use, `lo.Contains()` to check for membership.

Reply:
decided to go with a map. Since look up will be faster in case there are 100s or 1000s of indexes